repos/fdbt-site/cypress_tests/cypress/support/steps.ts

```diff
@@ -385,31 +385,13 @@ export const editServicesOtherProductsPage = (): void => {
 
 export const editProductName = () => {
     clickElementById('edit-product-name');
-    let oldProductName;
-    cy.get('.popup')
-        .find('[id=product-name]')
-        .should(($input) => {
-            oldProductName = $input.val();
-        });
-    const newProductName = Math.random().toString(36).substring(2, 7);
-    if (oldProductName === newProductName) {
-        clickElementByText('Cancel');
-    } else {
-        cy.get('.popup').find('[id=product-name]').clear().type(newProductName);
-        clickElementByText('Save');
-    }
-    cy.get('[id=product-name-header]').should('have.text', newProductName);
+    const newProductName = `Product ${getRandomNumber(1, 200)}${getRandomNumber(1, 200)}${getRandomNumber(1, 200)}`;
+    cy.get('.popup').find('[id=product-name]').clear().type(newProductName);
```

Comment:
getElementById('product-name') .clear()...

can we use the above line

Reply:
Im unsure because its a .find() inside a .get(), will this work for a popup?